packages/kbn-ai-playground/components/open_ai_key_callout.tsx

```diff
@@ -0,0 +1,49 @@
+import { EuiButton, EuiCallOut } from "@elastic/eui";
+import { css } from "@emotion/react";
+import { i18n } from "@kbn/i18n";
+
+import React, { Dispatch, SetStateAction } from "react";
+
+interface ListProps {
+  setIsOpenAIFlyOutOpen: Dispatch<SetStateAction<boolean>>;
+}
+
+export const OpenAIKeyCallout: React.FC<ListProps> = ({ setIsOpenAIFlyOutOpen }: ListProps) => {
```

Comment:
nitpick: as it's a js arrow function and we don't use any body we can reduce code by remove return.
```
({ setIsOpenAIFlyOutOpen }: ListProps) => {
<EuiCallout 
...
/>
)
```

Reply:
I have chosen this format as we will need to add more code eventually. However, as we do not need now, I agree that we can remove it for now.